e2e/src/api/specs/library.e2e-spec.ts

```diff
@@ -467,7 +454,7 @@ describe(`${LibraryController.name} (e2e)`, () => {
         expect(pathResponse).toEqual({
           importPath: pathToTest,
           isValid: false,
-          message: `Path does not exist (ENOENT)`,
+          message: `Not a directory`,
```

Comment:
It looks like this nest was never actually implemented so I've gone ahead and fixed it.